src/main/java/no/unit/marc/SearchRetrieveResponseParser.java

```diff
@@ -89,7 +90,7 @@
         }
     }
 
-    private static List<Document> getMarcFriendlyDocuments(String xml)
+    public static List<Document> getMarcFriendlyDocuments(String xml)
```

Comment:
Why from private to public? For testing ville være protected nok.

Reply:
Det har du nok rett i, skal endre